src/bindings.coffee

```diff
@@ -156,6 +156,9 @@ class Rivets.Binding
     else
       Rivets.Util.getInputValue el
 
+  attributeName: =>
+    @attrName or= [@view.prefix, @type].join('-').replace '--', '-'
```

Comment:
As I've mentioned before I'm no coffeescript expert but it looks like you're caching the attribute name under `attrName` as a property on the `Binding` instance, is that right? Feels like pollution to add both `attributeName` and `attrName` to all `Binding` instances just to save some code duplication, could you use a closure for caching the value instead?